packages/ruleset-migrator/src/requireResolve.ts

```diff
@@ -1,6 +1,6 @@
 export default <NodeRequire['resolve'] | null>((id, opts) => {
   try {
-    return require.resolve(id, opts);
+    return eval('require.resolve')(id, opts);
```

Comment:
could we use `module` in the same as we do it [here](https://github.com/stoplightio/spectral/blob/97134a5fb37b8a1c9cb00cccbcf68fcb7bd9a959/packages/ruleset-bundler/src/loader/node.ts#L1) and [here](https://github.com/stoplightio/spectral/blob/97134a5fb37b8a1c9cb00cccbcf68fcb7bd9a959/packages/cli/src/services/linter/utils/getRuleset.ts#L6).

You can provide `process.cwd()` as the base uri.

Reply:
I hope I did it correctly 😅

Reply:
looks good!
We have some tests that cover it so if something was off, they would fail.